packages/react/src/LayoutExamples/KitchenSink.tsx

```diff
@@ -230,6 +230,59 @@ export function KitchenSink(props: KitchenSinkProps) {
             </FAQ.Answer>
           </FAQ.Item>
         </FAQ>
+        {/* TODO: Figure out what should stay and go here */}
+        {/* Adding a className to stack causes vertical direction */}
+        {/* The second trick might be applied to Stack by default */}
+        <div className="FeatureTrio">
+          <Stack direction="horizontal" padding="spacious" gap="spacious" style={{textAlign: 'center'}}>
+            <style>
+              {`
+			  	.FeatureTrio {
+					margin: 100px auto;
+				}
+
+				.FeatureTrio img {
+					box-shadow: var(--brand-River-visual-shadow);
```

Comment:
Don't think our example needs to have a box shadow. Recommend we remove this. cc. @danielguillan.

Reply:
I have a note for our meeting on this as well but without one the color is pretty light and hard to see on some monitors. (like mine)